package.json

```diff
@@ -31,8 +31,8 @@
         "@sa11y/jest": "^0.1.4-beta.0",
```

Comment:
I don't think that you have the latest versions in your dev dependencies.
For comparison, here's what I have on Automation Compoents:
```
"@prettier/plugin-xml": "^0.13.0",
"@sa11y/jest": "^0.2.1",
"@salesforce/eslint-config-lwc": "^0.9.0",
"@salesforce/eslint-plugin-aura": "^1.4.0",
"@salesforce/sfdx-lwc-jest": "^0.10.4",
"eslint": "^7.19.0",
"eslint-config-prettier": "^7.2.0",
"husky": "^4.3.8",
"lint-staged": "^10.5.3",
"prettier": "^2.2.1",
"prettier-plugin-apex": "^1.8.0",
"semver": "^7.3.4"
```